test/generalized_package/setup.cfg

```diff
@@ -0,0 +1,2 @@
+[bdist_wheel]
```

Comment:
This generalized package is introduced to:
- to prevent that its not being built by `make test` and used by mistake in `nexus-integration` tests
- to avoid duplication and allow easy override for the package name and version directly from the tests. See `test_fail_to_upload_if_package_version_not_pep440_compliant`. This will allow later to add additional tests with the same package files.